src/main/resources/assets/community_mod/lang/en_us.lang

```diff
@@ -67,3 +67,17 @@ entity.community_mod.exploding_chicken.name=Exploding Chicken
 
 entity.community_mod.wood_gnome.name=Wood Gnome
 tile.antcake.full_of_ants=Heck no, it's full of ants
+
+#DavidM
+tile.community_mod.meme_altar.name=Meme Altar
+tooltip.community_mod.meme_altar_1=Right click to insert item.
+tooltip.community_mod.meme_altar_2=Right click again to extract item.
+tooltip.community_mod.altar_item=Put this on the Meme Altar to:
```

Comment:
Took me ages to figure out why there is "Electric Boolagoo" at the end tooltips ;)